docs/self-managed/setup/upgrade.md

```diff
@@ -27,6 +27,7 @@ Review the Camunda 8 Helm chart [version matrix](https://helm.camunda.io/camunda
 You can also view all chart versions and application versions via the Helm CLI:
 
 ```shell
+helm repo update
```

Comment:
Let's be explicit and add that command before installing and upgrading to avoid user mistakes.

Reply:
There is no `helm install` command on this page. I added the `helm repo update` command before the `helm upgrade` commands on this page.